src/runtime/lib/core/composables/use-meta-tracker-store.ts

```
@@ -37,13 +42,34 @@ export function updateMetaTracker(config: UpdateMetaTrackerConfig) {
+  const formKey
+    = typeof basePath === "string"
+      ? metaTracker[basePath]?.formKey ?? null
+      : null
+  if (formKey === null) {
+    throw new Error(`Unexpected Error: formKey is not defined
+- metaTracker passed to \`updateMetaTracker\` at base '${basePath ?? ""}'`)
+  }
 
   const metaTrackerPatch = Object.entries(flattenedObject).reduce<MetaTracker>(
     (acc, [key, value]) => ({
       ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)